### ISSUE TITLE
Add in SLP functionalities to CLI

### ISSUE THREAD

**Issue by lightswarm124** [Bitcoin-com/bch-cli-wallet#16]
I'm thinking it would be useful to include the SLP functionalities, such as create and mint token, into the CLI commands. There's a few examples on the SPL-SDK repo, although those are geared towards more of a pure node.js environment

**Comment by christroutner** [comment]
Yes, SLP integration is on the Roadmap in #4. I created the examples in the SLP-SDK repo, and I plan to porting them into bch-cli-wallet when I can find time. If you want to take a stab at it, I'd love to review a PR.
